src/importer/Program.cs

```diff
@@ -11,36 +11,35 @@ static void Main(string[] args)
         {
             var app = new CommandLineApplication();
             var folderOption = app.Option("-$|-f|--folder <folder>", "Folder to read UCAS .xls files from ", CommandOptionType.SingleValue);
+            var apiOption = app.Option("-u|--url|--api-url <url>", "URL of the ManageCourses API", CommandOptionType.SingleValue);
+            var apiKeyOption = app.Option("-k|--key|--api-key <key>", "Admin Key for the ManageCourses API", CommandOptionType.SingleValue);
+            
             app.HelpOption("-?|-h|--help");
             app.Execute(args);
 
-            var users = new CsvReader().ReadUsers(folderOption.Value());
-            var organisations = new CsvReader().ReadOrganisations(folderOption.Value());
-            var organisationInstitutions = new CsvReader().ReadOrganisationInstitutions(folderOption.Value(), organisations);
-            var organisationUsers = new CsvReader().ReadOrganisationUsers(folderOption.Value(), organisations, users);
+            // only used to avoid importing orphaned campuses
```

Comment:
?

Reply:
❗️

Reply:
More helpfully, we don't import institutions. We only use them to skip campuses with unknown institution codes (that's from earlier work by tim)

Reply:
Cool that makes more sense